website/docs/addon_applications_admin.md

```diff
@@ -30,6 +33,22 @@ The Addon features two primary settings categories: Definitions and Filters:
 1. **Definitions:** available only in studio settings, allows to define all possible applications and tools that can be used across all projects.
 2. **Filters:** defines when certain applications and tools are used on project level. This filters whether a certain application or tool should be enabled in a certain launch context, like a specific task type or project-wide.
 
+
+### Applications Web Actions
+
+:::info
+The web actions feature requires AYON Server version `1.3.0` or higher, core addon version `0.4.4` or higher, and Launcher version `1.1.0` or higher.
+
+Web actions run through **shims**, which are automatically set up when users install the AYON launcher version `1.1.0` or higher.
+:::
+
+Web actions let you launch applications directly through the AYON web server. You can find these actions in the [details panel](artist_details_panel.md).
+
+The application addon handles the logic for web actions.Currently, there are no specific settings for web actions.
+
+Web actions are generated from your application's definitions in your production or development bundles, depending on the developer mode setting, and they adhere to your application’s filter settings.
```

Comment:
```suggestion
The application web actions are generated from your application's definitions in your production or development bundles, depending on the developer mode setting, and they adhere to your application’s filter settings.
```
I think it may be nice to be extra clear here once more that we're only talking about the web actions for applications.

Reply:
Thank you https://github.com/ynput/ayon-documentation/pull/287/commits/970a51c94d067dbd8e5c09e153f0b83387eb2fb2